crates/ruma-client-api/src/dehydrated_device/get_events.rs

```diff
@@ -0,0 +1,58 @@
+//! `GET /_matrix/client/*/dehydrated_device/{device_id}/events`
+//!
+//! Get to-device events for a dehydrated device.
+
+pub mod unstable {
+    //! `msc3814` ([MSC])
+    //!
+    //! [MSC]: https://github.com/uhoreg/matrix-doc/blob/shrivelled_sessions/proposals/3814-dehydrated-devices-with-ssss.md
+
+    use ruma_common::{
+        api::{request, response, Metadata},
+        events::AnyToDeviceEvent,
+        metadata,
+        serde::Raw,
+        OwnedDeviceId,
+    };
+
+    const METADATA: Metadata = metadata! {
+        method: POST,
+        rate_limited: false,
+        authentication: AccessToken,
+        history: {
+            unstable => "/_matrix/client/unstable/org.matrix.msc3814.v1/dehydrated_device/:device_id/events",
+        }
+    };
+
+    /// Request type for the `dehydrated_device/{device_id}/events` endpoint.
+    #[request(error = crate::Error)]
+    pub struct Request {
+        /// The unique ID of the device for which we would like to fetch events for.
```

Comment:
```suggestion
        /// The unique ID of the device for which we would like to fetch events.
```
We need only one of the `for`s :wink:.